### PR TITLE
Show drive space usage

### PR DESCRIPTION
**Pull request by emveepee**
Optionally show drive space by default drive or by summing all drives

### CODE REVIEW

File: src/Recordings.cpp [hunk 1]
```diff
@@ -119,6 +177,11 @@ PVR_ERROR Recordings::GetRecordings(bool deleted, kodi::addon::PVRRecordingsResu
       }
     }
     m_iRecordingCount = recordingCount;
+    // force read disk space
+    m_checkedSpace = 0;
+    uint64_t total;
+    uint64_t used;
+    GetDriveSpace(total, used);
```
Comment (phunkyfish) [inline src/Recordings.cpp:184]: Why have this here?

Comment (emveepee) [reply src/Recordings.cpp:184]: > 
> Why have this here?

The check to the backend is every 5 minutes.  This recording list update is triggered right after a user does maintenance so by forcing the update, the user will get dynamic updates as shows are removed and timers scheduled to provide this information in a timely manner.

Comment (phunkyfish) [reply src/Recordings.cpp:184]: Cool, makes sense.

**Review by phunkyfish — APPROVED** [review]
LGTM

**Pull request merged**